server/commands/list_objects.go

```diff
@@ -0,0 +1,279 @@
+package commands
+
+import (
+	"context"
+	"sync/atomic"
+	"time"
+
+	"github.com/go-errors/errors"
+	"github.com/openfga/openfga/pkg/logger"
+	"github.com/openfga/openfga/pkg/tuple"
+	serverErrors "github.com/openfga/openfga/server/errors"
+	"github.com/openfga/openfga/storage"
+	openfgapb "go.buf.build/openfga/go/openfga/api/openfga/v1"
+	"go.opentelemetry.io/otel/metric"
+	"go.opentelemetry.io/otel/metric/instrument"
+	"go.opentelemetry.io/otel/metric/unit"
+	"go.opentelemetry.io/otel/trace"
+	"golang.org/x/sync/errgroup"
+)
+
+const (
+	MaximumConcurrentChecks = 100 // todo(jon-whit): make this configurable, but for now limit to 100 concurrent checks
```

Comment:
I dont' know that this needs to be a publicly exported symbol. It's only relevant within the scope of this package. Let's change it to `maximumConcurrentCheck`.